webapp/components/instantiation/NetPyNEInstantiated.js

```diff
@@ -141,6 +141,7 @@ class NetPyNEInstantiated extends React.Component {
               : (this.props.theme === THEMES.LIGHT ? canvasBgLight : bgRegular)
           }
           onSelection={this.onSelection}
+          linesThreshold="5000"
```

Comment:
When I was working on the [`linesThreshold` issue](https://app.zenhub.com/workspaces/geppetto-meta-6047b13d88e7e70013d4589b/issues/metacell/geppetto-meta/347) , I noticed that setting the linesThreshold to 10,000 solved the problem. I tested it on NetPyNE's development as well as this branch and it worked.

Reply:
fixed now! Thx